packages/web-components/@sfx/ui/stories/index.ts

```diff
@@ -17,19 +18,51 @@ const items = [
   }
 ];
 
+function getListComponent(codeSnippet: boolean = false): string {
+  const title = text('SF-X List Title', 'Items');
+  const list = text('List Items', JSON.stringify(items));
+
+  return codeSnippet === false
+    ? '<sfx-list\n' + ` caption="${title}"\n` + ` items="${list}"\n` + '></sfx-list>'
+    : '<sfx-list\n' + ` caption="${title}"\n` + '></sfx-list>';
+}
+
 storiesOf('UI|List', module)
   .addDecorator(withKnobs)
   .add(
-    'Default',
-    () => `
-      <sfx-list items="${text('List Items', JSON.stringify(items))}"></sfx-list>
-    `
-  )
-  .add(
-    'With Title',
-    () => `
-      <sfx-list 
-        caption="${text('Sfx List Title', 'Items')}" 
-        items="${text('List Items', JSON.stringify(items))}"></sfx-list>
-    `
+    'Rendering with data populated via attributes',
+    () => {
+      return `
+      ${getListComponent()}
+      ${getDisplayCode(getListComponent(true))}
+    `;
+    },
+    {
+      notes: {
+        markdown: `# SF-X List Component
+
+        [SF-X UI README](https://github.com/groupby/sfx-view/tree/master/packages/web-components/%40sfx/ui "SF-X List README")
+
+        \`\`\`html
+        <sfx-ui
+          caption="Items"
+          list=[]
+        ></sfx-ui>
+        \`\`\`
+
+        ## Demonstrated in this story:
+
+          * The SF-X List component rendering a list with data populated via the 'caption' and 'list' attributes.
```

Comment:
Again, I think the subpoints should reworded and promoted, but to fix this line as-is: 

```suggestion
          * The SF-X List component renders a list with data from the \`caption\` and \`list\` attributes.
```